packages/core/src/components/discord-poll-answer/DiscordPollAnswer.ts

```diff
@@ -0,0 +1,463 @@
+		}
+
+		:host([light-theme]) .discord-background-color-default {
+			background-color: color-mix(in oklab, hsl(223 calc(1 * 5.8%) 52.9% / 0.2) 100%, hsl(0 0% 0% / 0.2) 0%) !important;
+		}
+
+		:host([light-theme]) .discord-awnswer-title,
+		:host([light-theme]) .discord-quantity-votes,
+		:host([light-theme]) .discord-percentage-votes {
+			color: color-mix(in oklab, hsl(223 calc(1 * 6.7%) 20.6% / 1) 100%, black 0%) !important;
+		}
+
+		:host([light-theme]) .discord-checkbox-div-answer,
+		:host([light-theme]) .discord-checkbox-div-multiple-answer {
+			border-color: color-mix(in oklab, hsl(223 calc(1 * 6.7%) 20.6% / 1) 100%, black 0%) !important;
+		}
+
+		:host([light-theme]) .discord-checkbox-div-answer-selected-ended-no-winner {
+			background-color: color-mix(in oklab, hsl(223 calc(1 * 6.7%) 20.6% / 1) 100%, black 0%);
+			color: white !important;
+		}
+
+		.discord-answer-no-margin {
+			margin: 0;
+		}
+
+		.discord-quantity-votes:hover {
+			cursor: pointer;
+			text-decoration: underline;
+		}
+
+		.discord-answer-backdround-color {
+			content: '';
+			position: absolute;
+			height: 100%;
+			left: 0;
+			z-index: -1;
+		}
+
+		.discord-background-color-winner {
+			background-color: color-mix(in oklab, hsl(145 calc(1 * 65%) 39.2% / 0.2) 100%, hsl(0 0% 0% / 0.2) 0%);
+		}
+
+		.discord-background-color-default {
+			background-color: color-mix(in oklab, hsl(228 calc(1 * 6%) 32.5% / 0.48) 100%, hsl(0 0% 0% / 0.48) 0%);
+		}
+
+		.discord-background-color-selected {
+			background-color: color-mix(in oklab, hsl(235 calc(1 * 85.6%) 64.7% / 0.2) 100%, hsl(0 0% 0% / 0.2) 0%);
+		}
+	`;
+
+	@property({ type: String, attribute: 'emoji', reflect: true })
+	public accessor emoji: string;
+
+	@property({ type: String, attribute: 'answer', reflect: true })
+	public accessor answer: string;
+
+	@property({ type: Number, attribute: 'votes', reflect: true })
+	public accessor votes: number = 0;
+
+	@property({ type: Boolean, attribute: 'selected', reflect: true })
+	public accessor selected: boolean = false;
+
+	/**
+	 * Whether to use compact mode or not.
+	 */
+	@consume({ context: messagesCompactMode })
+	public accessor compactMode = false;
+
+	@consume({ context: multipleAnswersPoll })
+	public accessor multipleAnswers = false;
+
+	@consume({ context: pollEnded })
+	public accessor pollEnded = false;
+
+	@consume({ context: pollVoted })
+	public accessor pollVoted = false;
+
+	@consume({ context: showResults })
+	public accessor showResult = false;
+
+	/**
+	 * Whether to use ligth theme or not.
+	 */
+	@consume({ context: messagesLightTheme })
+	@property({ type: Boolean, reflect: true, attribute: 'light-theme' })
+	public accessor lightTheme = false;
+
+	private accessor totalVotesPoll = 0;
+
+	private accessor percentageVoted: number;
+
+	private accessor arrayAnswers: any[] = [];
+
+	private accessor winners: any[] = [];
```

Comment:
Please refactor out these anys and give them proper types. Right now this is very bug prone and I suspect there might even be bugs right now.

On line 258 the type of `arrayAnswers` would be `{ answer: string; value: number }[]` and from line 266 `winners` would be `string[]` but then line 265 will fail because `string[]` will never include `this.arrayAnswers[index]` (it might include `this.arrayAnswers[index].answer` though)